src/scripts/main.js

```diff
@@ -1 +1,73 @@
 'use strict';
+
+// First promise
+const firstPromise = new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject(new Error('First promise was rejected in 3 seconds if not clicked'));
+  }, 3000);
+
+  document.addEventListener('click', (evnt) => {
+    if (evnt.button === 0) {
+      clearTimeout(timeoutId);
+      resolve('First promise was resolved on a left click in the document');
+    }
+  });
+});
+
+// Second promise
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', (evnt) => {
+    if (evnt.button === 0 || evnt.button === 2) {
```

Comment:
same. click event not works for right button
try to listen for right button `contextmenu` or `mousedown` event

Reply:
still not fixed